extensions-contrib/kafka-emitter/src/main/java/org/apache/druid/emitter/kafka/KafkaEmitterConfig.java

```diff
@@ -47,19 +74,76 @@
   @JsonCreator
   public KafkaEmitterConfig(
       @JsonProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG) String bootstrapServers,
-      @JsonProperty("metric.topic") String metricTopic,
-      @JsonProperty("alert.topic") String alertTopic,
+      @Nullable @JsonProperty("event.types") Set<String> eventTypes,
+      @Nullable @JsonProperty("metric.topic") String metricTopic,
+      @Nullable @JsonProperty("alert.topic") String alertTopic,
       @Nullable @JsonProperty("request.topic") String requestTopic,
+      @Nullable @JsonProperty("segment.metadata.topic") String segmentMetadataTopic,
+      @Nullable @JsonProperty("segment.metadata.topic.format") String segmentMetadataTopicFormat,
```

Comment:
same comment about de-serializing directly to SegmentMetadataTopicFormat instead of going through String values and de-serializing manually in the emitter.